src/libsndfile-1.def

```diff
@@ -1,6 +1,4 @@
-; Auto-generated by cmake
-
-LIBRARY ${PROJECT_NAME}-${PROJECT_VERSION_MAJOR}.dll
+LIBRARY libsndfile-1.dll
```

Comment:
Is `LIBRARY ${PROJECT_NAME}-${PROJECT_VERSION_MAJOR}.dll` not correct? I think it is. I don't see any reason to make this change (or the similar one above).